addons/loyalty/views/loyalty_rule_views.xml

```diff
@@ -37,8 +36,9 @@
                                 <field name="reward_point_amount" class="oe_inline me-1"/>
                                 <field name="reward_point_name" class="w-auto"/>
                             </div>
-                            <label for="reward_point_mode" string=""/>
-                            <field name="reward_point_mode" widget="radio" nolabel="1" invisible="program_type not in ('promotion', 'promo_code', 'next_order_coupons', 'loyalty', 'buy_x_get_y')"/>     
+                            <field colspan="2" class="d-flex" name="reward_point_mode" widget="radio" nolabel="1"
+                                invisible="program_type not in ('promotion', 'promo_code', 'next_order_coupons', 'loyalty', 'buy_x_get_y')"
+                            />
```

Comment:
```suggestion
                            <field name="reward_point_mode" 
                                   colspan="2"
                                   class="d-flex"
                                   widget="radio"
                                   nolabel="1"
                                   invisible="program_type not in ('promotion', 'promo_code', 'next_order_coupons', 'loyalty', 'buy_x_get_y')"
                            />
```
Keep the name as the first attribute. Either keep all attributes in the same line or one attribute one line. Remember about proper indentation

Reply:
Thanks, it's prettier and easier to read :)